### PR TITLE
Snackbar component

### PR DESCRIPTION
**Pull request by roblevintennis**
**Description:**

- [Asana Task](https://app.asana.com/0/1150068311310825/1163005231692857/f)
- EDS Snackbar component

- [x] Have exported the component from the main [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [x] Have you exported it from the [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [x] Bumped the yarn version?

### CODE REVIEW

File: src/components/Snackbar/Snackbar.module.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import '../Animation.scss';
```
Comment (roblevintennis) [inline src/components/Snackbar/Snackbar.module.scss]: For the Nora component, we'll basically reimplement this to skin as Tom's designed things

File: src/components/Animatable.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+.ZoomInUp {
+  animation-name: zoomInUp;
+}
+
+@keyframes zoomInUp {
+  from {
+    opacity: 0;
+    transform: scale3d(0.5, 0.5, 0.5) translate3d(0, 1000px, 0);
+    animation-timing-function: cubic-bezier(0.55, 0.055, 0.675, 0.19);
```
Comment (roblevintennis) [inline src/components/Animatable.scss]: These are pretty specific at the moment, and so I've elected to not share them yet in Animation.scss. If I start to see a repeatable pattern I will, but for now this is one-off

**Comment by roblevintennis** [comment]
In working on this in tandem with Nora, I discovered that since Nora uses Nest.js, we have to account for both the Fetch API Response interface, and also the Nest.js one—Nest.js uses `statusCode` not `status` for example.

Here's manually testing after changes made, and the XHR examples all appear to still work properly:

image

File: src/nora/components/NoraTextAreaInput/NoraTextAreaInputExample.module.scss [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 .TextClassExample {
   margin-top: var(--Space-12);
-  font-size: calc(var(--compact-font-size)* 1px);
+  font-size: var(--Nora-Font-15);
```
Comment (stephaniel96) [inline src/nora/components/NoraTextAreaInput/NoraTextAreaInputExample.module.scss]: 👍

**Pull request merged**